tools/generate.cmd

```diff
@@ -6,6 +6,22 @@
 @echo off
 setlocal
 
+:: help requested?
+set pot_help="%1%2"
+if not x%pot_help%==x%pot_help:help=% (set req_help=T)
+if not x%pot_help%==x%pot_help:?=%    (set req_help=T)
+if not "%req_help%" == "" (
```

Comment:
You are using some awesome cmd logic that I don't understand. What is the syntax for help here? Seems like `generate help` || `generate ?`, is that right?